denys/Lesson01-Favorite-Toys/T01.01-Exercise-CreateLayout/app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+          package="com.example.android.favoritetoys">
+
+    <application
+        android:allowBackup="true"
```

Comment:
My immediate thought was yes simply because `allowBackup=true` comes with a smaller security risk. So is there any reason to opt in on it before we'd need it?